package java16;

import java.io.File;
import java.io.FileNotFoundException;
import java.util.Scanner;

public class TestTryWithResourcesCatchJ16 {
   public void test(File var1) {
      try (Scanner var2 = new Scanner(var1)) {// 9
         var2.next();// 10
      } catch (FileNotFoundException var7) {// 11
         var7.printStackTrace();// 12
      }

   }// 14

   public void testFunc(File var1) {
      try (Scanner var2 = this.create(var1)) {// 17
         var2.next();// 18
      } catch (FileNotFoundException var7) {
         var7.printStackTrace();// 20
      }
   }// 22

   private Scanner create(File var1) throws FileNotFoundException {
      return new Scanner(var1);// 25
   }
}

class 'java16/TestTryWithResourcesCatchJ16' {
   method 'test (Ljava/io/File;)V' {
      4      8
      8      8
      9      9
      a      9
      b      9
      c      9
      2a      10
      2c      11
      2f      14
   }

   method 'testFunc (Ljava/io/File;)V' {
      0      17
      1      17
      2      17
      3      17
      4      17
      5      17
      6      18
      7      18
      8      18
      9      18
      31      20
      34      22
   }

   method 'create (Ljava/io/File;)Ljava/util/Scanner;' {
      4      25
      8      25
   }
}

Lines mapping:
9 <-> 9
10 <-> 10
11 <-> 11
12 <-> 12
14 <-> 15
17 <-> 18
18 <-> 19
20 <-> 21
22 <-> 23
25 <-> 26
Not mapped:
13
19
21
